src/plugins/data/common/data_frames/utils.ts

```diff
@@ -290,6 +290,26 @@ export const getTimeField = (
     : fields.find((field) => field.type === 'date');
 };
 
+/**
+ * Parses timepicker datetimes using datemath package. Will attempt to parse strings such as
+ * "now - 15m"
+ *
+ * @param dateRange - of type TimeRange
+ * @returns object with fromDate and toDate, both of which will be in utc time and formatted to
+ * 'YYYY-MM-DD HH:mm:ss.SSS'
+ */
+export const formatTimePickerDate = (dateRange: TimeRange) => {
```

Comment:
Hi @paulstn, can we add some unit tests for this function ? 

Also it is missing the changelog entry in the PR description.

Reply:
I was able to get the changelog bot working, its updated now

Reply:
updated with tests